py/private/toolchain/python.sh

```diff
@@ -9,7 +9,7 @@ PYTHON_BIN="{{PYTHON_BIN}}"
 if [ -z "${VIRTUAL_ENV:-}" ]; then
   exec "${PYTHON_BIN}" "$@"
 else
-  PYTHON_REAL="${VIRTUAL_ENV}/bin/python_real"
+  PYTHON_REAL="${VIRTUAL_ENV}/bin/python3"
```

Comment:
some comments or test cases so we know why it has to be this way? (similar to the commit description?)